src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -17,7 +23,19 @@
  * Результат 2: acf</p>
  */
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    public String getUniqueCharacters(String fileName) throws IOException {
+        String fileText = Files.readString(Paths.get(fileName));
+        StringBuilder sb = new StringBuilder();
+        TreeSet<String> text = new TreeSet<>(Arrays.asList(fileText.replaceAll("[^a-zA-Z]", "")
```

Comment:
better using Character than String, and methods Character.isLetter(), Character.toLowerCase()

Reply:
ok